### PR TITLE
resolveReleasesFromAddress -> resolveContractUriFromAddress

### PR DESCRIPTION
**Pull request by nachoiacovino**

### CODE REVIEW

File: packages/sdk/src/evm/common/feature-detection.ts [hunk 1]
```diff
@@ -246,6 +246,7 @@ export async function resolveContractUriFromAddress(
   const bytecode = await provider.getCode(address);
   if (bytecode === "0x") {
     const chain = await provider.getNetwork();
+    console.log({address, provider, chain});
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/common/feature-detection.ts:249]: remove log?

File: packages/sdk/src/evm/core/classes/contract-publisher.ts [hunk 2]
```diff
@@ -195,13 +195,13 @@ export class ContractPublisher extends RPCConnectionHandler {
...
    */
-  public async resolveReleasesFromAddress(address: string) {
+  public async resolveContractUriFromAddress(address: string) {
     const contractUri = await resolveContractUriFromAddress(
       address,
       this.getProvider(),
     );
     invariant(contractUri, "Could not resolve contract URI from address");
-    return await this.resolvePublishMetadataFromCompilerMetadata(contractUri);
+    return contractUri;
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/core/classes/contract-publisher.ts:204]: can you summarize what the problem was @nachoiacovino ?

Comment (nachoiacovino) [reply packages/sdk/src/evm/core/classes/contract-publisher.ts:204]: we weren't passing the provider as before so we were hitting the provider from polygon always instead of hitting the provider from the deployed contract for getting contractUri, so now we're calling this function with the deployed contract chainId then getting releases with the polygon sdk

Comment (joaquim-verges) [reply packages/sdk/src/evm/core/classes/contract-publisher.ts:204]: got it that makes sense

**Review by joaquim-verges — APPROVED** [review]
